tests/test_sipApproximation.py

```diff
@@ -561,8 +561,15 @@ def run(original, width, height, order, spacing=20, skyTol=1.0e-4, pixTol=1.0e-8
             # Evaluate performance
             points = [Point2D(xx, yy) for xx, yy in itertools.product(range(0, width, spacing),
                                                                       range(0, height, spacing))]
```

Comment:
Should this be something like
```
x_points, y_points = np.meshgrid(range(0, width, spacing), range(0, height, spacing))
x_points = x_points.ravel()
y_points = y_points.ravel()
```
And I guess we still need the spherepoint conversion because of the `getTangentPlaneOffset` call.

Reply:
I changed to meshgrid. The number of elements is the same but they come out in a different order -- that won't change the test.